doc-content/enterprise-only/springboot/bus-app-rh-sso_proc.adoc

```diff
@@ -1,23 +1,25 @@
 [id='bus-app-rh-sso_{context}']
 = Configuring the business application with Red Hat Single Sign-On
 
-You can use Red Hat Single Sign-On (RH SSO) to enable single sign-on between your services and to have a central place to configure and manage your users and roles.
+Most organizations provide user and group details through single sign-on (SSO) tokens. You can use Red Hat Single Sign-On (RHSSO) to enable single sign-on between your services and to have a central place to configure and manage your users and roles.
 
 .Prerequisites
-* You have a `<business-application>.zip` file that you created using the http://start.jbpm.org[business applications] website.
+* You have a {PRODUCT} Spring Boot application that you created using the http://start.jbpm.org[business applications] website.
 
 .Procedure
-. Download and install RH SSO. For instructions, see the https://access.redhat.com/documentation/en-us/red_hat_single_sign-on/{RH-SSO_VERSION}/html/getting_started_guide/[_Red Hat Single Sign-On Getting Started Guide_].
-. Configure RH SSO:
+. Download and install RHSSO. For instructions, see the https://access.redhat.com/documentation/en-us/red_hat_single_sign-on/{RH-SSO_VERSION}/html/getting_started_guide/[_Red Hat Single Sign-On Getting Started Guide_].
+. Configure RHSSO:
 .. Either use the default master realm or create a new realm.
++
+A Realm manages a set of users, credentials, roles, and groups. A user belongs to and logs into a realm. Realms are isolated from one another and can only manage and authenticate the users that they control.
 .. Create the `springboot-app` client and set the `AccessType` to public.
 .. Set a valid redirect URI and web origin according to your local setup, for example:
 +
 * Valid redirect URIs: `\http://localhost:8090/*`
 * Web origin: `\http://localhost:8090`
 .. Create realm roles that are used in the application.
 .. Create users that are used in the application and assign roles to them.
-. Add the following dependencies to the service project `pom.xml` file:
+. Add the following dependencies to the Spring Boot project `pom.xml` file:
```

Comment:
project's?

Add the following dependencies to the Spring Boot project's `pom.xml` file: